setup.py

```diff
@@ -34,9 +34,14 @@ def parse_requirements_file(path):
     install_requires=parse_requirements_file('requirements.txt'),
     test_suite='acceptable.tests',
     tests_require=parse_requirements_file('requirements-dev.txt'),
+    include_package_data=True,
+    package_data={
+        'acceptable': ['templates/*'],
+    },
```

Comment:
Sure, it's not a new problem, it's making an existing problem worse. I'm fine if you want to leave this as-is in this PR, but I think it's worth fixing sooner rather than later.